### ISSUE TITLE
UnicodeDecodeError: 'charmap' codec can't decode byte 0x9d in position 713: character maps to <undefined>

### ISSUE THREAD

**Issue by jamesgecko** [lektor/lektor#471]
I've run into an issue very similar (if not identical) to the closed #378. I'm running the latest version of Windows 10.

First I ran the command on the website in Powershell.
```
iex ((new-object net.webclient).DownloadString('https://getlektor.com/install.ps1'))
```

I didn't have Python installed, and `iex` closed the terminal immediately without feedback; I had to split it in two to see what was wrong.

```
PS C:\Users\james\Dropbox> (new-object net.webclient).DownloadString('https://getlektor.com/install.ps1') > install.ps1
...
PS C:\Users\james\Dropbox> ./install.ps1
```

The error said that I needed Python. I installed Python 3.6.3 to the default location in my user directory, `C:\Users\james\AppData\Local\Programs\Python\Python36-32\`

Then I ran the installation script again, this time without any errors. Then I ran quickstart.
```
PS C:\Users\james\Dropbox> lektor quickstart
Traceback (most recent call last):
  File "C:\Users\james\AppData\Local\Programs\Python\Python36-32\Lib\runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "C:\Users\james\AppData\Local\Programs\Python\Python36-32\Lib\runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "C:\Users\james\AppData\Local\lektor-cli\lib\Scripts\lektor.exe\__main__.py", line 5, in <module>
  File "c:\users\james\appdata\local\lektor-cli\lib\lib\site-packages\lektor\cli.py", line 8, in <module>
    from .i18n import get_default_lang, is_valid_language
  File "c:\users\james\appdata\local\lektor-cli\lib\lib\site-packages\lektor\i18n.py", line 17, in <module>
    translations[_lang] = json.load(f)
  File "C:\Users\james\AppData\Local\Programs\Python\Python36-32\Lib\json\__init__.py", line 296, in load
    return loads(fp.read(),
  File "c:\users\james\appdata\local\lektor-cli\lib\lib\encodings\cp1252.py", line 23, in decode
    return codecs.charmap_decode(input,self.errors,decoding_table)[0]
UnicodeDecodeError: 'charmap' codec can't decode byte 0x9d in position 713: character maps to <undefined>
```

**Comment by mitsuhiko** [comment]
This is a Python 3 specific bug that needs fixing. The call in i18n would need to open the file in binary mode.

**Comment by mitsuhiko** [comment]
Fixed by #378
